README.md

```diff
@@ -1,40 +1,108 @@
+For installing the plugin through nopCommerce, you can either upload the plugin directly from nopCommerce `Configuration > Local plugins > Upload plugin or a theme`
 
-To create the BTCPay WebHook, [read this](https://docs.btcpayserver.org/VirtueMart/#23-create-a-webhook-on-btcpay-server) and use the default secret code generated by BTCPay.
-- Note: Other than in the guide you need to copy the Url shown in field "Webhook Url" from your configuration screen on NopCommerce.
+Ensure the plugin file is in ZIP for successful upload. After uploading, find the plugin in the list, click "Install" next to it, and enable it if needed​.
+
+Alternatively you can manually add upload the plugin to the `/plugins` folder in your nopCommerce directory and restart the application.
+
+Once the plugin is uploaded, make sure to click `Install` button and `Restart application` to apply the changes.
+
+## Plugin configuration
+
+You can configure the plugin either automatically or manually. Automatic setup is much faster and recommended. To begin configuring the plugin, click on the `Configure` button in the
+
+### Automatic Configuration
+
+1. Enter Url to your BTCPay Server into **BTCPay URL** field. (e.g. https://mainnet.demo.btcpayserver.org)
+2. Click on the **Configure automatically** button to be redirected to the API authorization page of your BTCPay Server
+3. On the authorization page: Select the store you want to connect to your nopCommerce (you might need to login first)
+4. Click on "Authorize App" button and you will be redirected back to your nopCommerce
+5. The "API Key", "BTCPay Store ID" and "Webhook Secret" fields will be automatically filled and a webhook created
+6. Click **Save** button at bottom to persist the configuration. 
+7. Click **Activate** to enable the plugin. Congrats, the configuration is now complete!
+
+*Note: given the architecture of nopCommerce, it is possible that malfunctions in redirections may occur. Particularly if https is disabled or incorrectly configured. In this case, please opt for manual configuration.*
+
+### Manual Configuration
+
+Ensure that the following fields are filled out: 
+- BTCPay Url
+- BTCPay Store ID
+- API Key
+- WebHook Secret
+
+#### BTCPay URL
+
+In the BTCPay URL field of your nopCommerce, simply input the link to your self-hosted server or the one hosted by a [third-party provider](https://directory.btcpayserver.org/filter/hosts). For example; https://mainnet.demo.btcpayserver.org
+
+#### BTCPay Store ID
+
+BTCPay **Store ID** can be obtained from your BTCPay Server, in Store Settings > General > Store ID field. Copy it and paste it over in the `BTCPay Store ID` field.
+
+#### Generating the API key
+
+1. To create the **BTCPay API key**, click on the `Account` located at the bottom of the sidebar.
```

Comment:
maybe add " on your BTCPay Server" to make it clear that this action is done on BTCPay instance and not on nopCommerce

Reply:
Good one, fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e